src/PersonalInfo.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+class PersonalInfo extends React.Component {
+
+  constructor() {
+    super();
+    this.state = { name:'', bgColor:''}
+    this.onInputChange = this.onInputChange.bind(this);
+    this.validateField = this.validateField.bind(this);
+  }   
+
+  onInputChange(event) {
+    let name = event.target.name;
+    let value = event.target.value;
+    this.validateField(name, value);
+  }
+    
+  validateField(fieldName, value) {
+    let backgroungColor = this.state.bgColor;
```

Comment:
```suggestion
    let backgroundColor = this.state.backgroundColor;
```

whats `backgroungColor`? 😅